### ISSUE TITLE
Linking of 'To Do' with Stage'

### ISSUE THREAD

**Issue by CAKRA2017** [madrecha/jamku#1167]

**Comment by adarshmadrecha** [comment]
One what I am thinking this can be implemented:
1. We provide a field at Todo level which maps to task stage.
2. When all the todo in that stage is marked, it automatically completes that Stage.

Will this help in your use case @CAKRA2017 ?

**Comment by adarshmadrecha** [comment]
Also suggested by (In WhatsApp Jamku 3 Group)
1. Darshan Dagli
2. Yeshoshuva K

**Comment by mohdlak** [comment]
Yes , we have recommended this way back when we started using the software..
